src/test/java/net/eventsexpress/app/pages/footerpages/PrivacyPage.java

```diff
@@ -19,5 +19,9 @@ public PrivacyPage() {
     public String getPrivacyHeading() {
         return privacyHeading.getText();
     }
+    public static void assertPrivacyHeadingIsDisplayed() {
+        assert Base.findElementByCSS(PRIVACY_HEADING_CSS).isDisplayed()
```

Comment:
Use the @FindBy annotation, Base page is deprecated.

Reply:
Made new method that uses @findby annotation as requested